lib/index.js

```diff
@@ -65,10 +65,15 @@ Optimizely.prototype.initialize = function() {
 };
 
 /**
- * Track. The Optimizely Web trackEvent can only send one property, revenue
+ * Track. The Optimizely Classic Web trackEvent accepts a payload object.
  *
- * https://www.optimizely.com/docs/api#track-event
- * http://developers.optimizely.com/javascript/reference/#api-function-calls
+ * Optimizely Classic: https://developers.optimizely.com/classic/javascript/reference/index.html#api-function-calls
+ * Optimizely X:       https://developers.optimizely.com/x/solutions/javascript/reference/index.html#function_setevent
+ *
+ * This API is forward compatible from Optimizely Classic to Optimizely X.
+ *   - Classic will only utilize the revenue key of the payload
```

Comment:
Classic JS API will only inspect the revenue key.

It then forwards all API requests to the X JS API.

The X JS API will identify classic API calls and translate certain ones (where possible) to their functional equivalents in X.

In our case, the object being passed to the classic API call works for revenue only but is also passed along in its entirety to X.